liberty-maven-plugin/src/it/kernel-install-feature-test/prepare-multiple-features-it/pom.xml

```diff
@@ -0,0 +1,170 @@
+	                    <goals>
+	                        <goal>copy-resources</goal>
+	                    </goals>
+	
+	                    <configuration>
+			                <outputDirectory>${user.home}/.m2/repository/test/user/test/osgi/hello-esa-plugin/1.0</outputDirectory>
+			                <resources>
+			                    <resource>
+			                        <directory>src/test/resources</directory>
+			                        <includes>
+			                            <include>hello-esa-plugin-1.0.esa</include>
+			                        </includes>
+			                    </resource>
+			                </resources>
+			            </configuration>
+	                </execution>
+					<execution>
+	                    <id>copy-resource-three</id>
+	                    <phase>process-test-resources</phase>
+	                    <goals>
+	                        <goal>copy-resources</goal>
+	                    </goals>
+	
+	                    <configuration>
+			                <outputDirectory>${user.home}/.m2/repository/test/user/test/osgi/SimpleActivator-bom/1.0</outputDirectory>
+			                <resources>
+			                    <resource>
+			                        <directory>src/test/resources</directory>
+			                        <includes>
+			                            <include>SimpleActivator-bom-1.0.pom</include>
+			                        </includes>
+			                    </resource>
+			                </resources>
+			            </configuration>
+	                </execution>
+	                <execution>
+	                    <id>copy-resource-four</id>
+	                    <phase>process-test-resources</phase>
+	                    <goals>
+	                        <goal>copy-resources</goal>
+	                    </goals>
+	
+	                    <configuration>
+			                <outputDirectory>${user.home}/.m2/repository/test/user/test/osgi/SimpleActivatorESA/1.0</outputDirectory>
+			                <resources>
+			                    <resource>
+			                        <directory>src/test/resources</directory>
+			                        <includes>
+			                            <include>SimpleActivatorESA-1.0.esa</include>
+			                        </includes>
+			                    </resource>
+			                </resources>
+			            </configuration>
+	                </execution>
+	           </executions>
+	        </plugin>
+	        
+            <plugin>
+                <groupId>io.openliberty.tools</groupId>
+                <artifactId>liberty-maven-plugin</artifactId>
+                <version>@pom.version@</version>
+                <configuration>
+                    <assemblyArtifact>
+                        <groupId>${runtimeGroupId}</groupId>
+                        <artifactId>${runtimeKernelId}</artifactId>
+                        <!--  this goal is only available with ol 21.0.0.11 or later -->
+                        <version>21.0.0.11</version>
```

Comment:
We are currently running with 22.0.0.3 and 22.0.0.6 in our builds. Do we really need this hard coded to 21.0.0.11?

Reply:
No I don't think we need to hardcode this anymore. When this test was first created I think there were builds running on versions older than 21.0.0.11. I will update